src/latex/LaTeXWriter.jl

```diff
@@ -52,23 +52,26 @@ struct LaTeX <: Documenter.Writer
     platform::String
     version::String
     tectonic::Union{Cmd,String,Nothing}
+    code_listings::String
     function LaTeX(;
-            platform = "native",
-            version  = get(ENV, "TRAVIS_TAG", ""),
-            tectonic = nothing)
+        platform="native",
+        version=get(ENV, "TRAVIS_TAG", ""),
+        tectonic=nothing,
+        code_listings="minted")
+        code_listings ∈ ("minted", "listings") || throw(ArgumentError("unknown code formatting package: $platform"))
         platform ∈ ("native", "tectonic", "docker", "none") || throw(ArgumentError("unknown platform: $platform"))
-        return new(platform, string(version), tectonic)
+        return new(platform, string(version), tectonic, code_listings)
     end
 end
 
 import ..Documenter
 import Markdown
 import ANSIColoredPrinters
 
-mutable struct Context{I <: IO} <: IO
+mutable struct Context{I<:IO} <: IO
```

Comment:
Please no code reformatting (of course on `master` this is a no-issue as we use runic, i.e. `make format-julia` will ensure the code is formatted consistently.